ppr-ui/src/components/tables/SearchHistory.vue

```diff
@@ -58,29 +59,38 @@
                     </v-col>
                   </v-row>
                 </td>
-                <td v-if="isPprSearch(item)">
-                  {{ displayType(item.searchQuery.type) }}
-                </td>
-                <td v-else>
-                  {{ displayMhrType(item.searchQuery.type) }}
+                <td>
+                  <span class="aria-label-only"
+                    aria-hidden="false"
+                  >{{ headers[1].text }}</span>
+                  {{ isPprSearch(item) ? displayType(item.searchQuery.type) : displayMhrType(item.searchQuery.type) }}
                 </td>
                 <td>
+                  <span class="aria-label-only"
+                    aria-hidden="false"
+                  >{{ headers[2].text }}</span>
                   <span v-if="isPprSearch(item)">Personal Property</span>
                   <span v-else>Manufactured Homes</span>
                 </td>
                 <td>
+                  <span class="aria-label-only"
+                    aria-hidden="false"
+                  >{{ headers[3].text }}</span>
                   <span v-if="!item.inProgress || isSearchOwner(item)">
                     {{ displayDate(item.searchDateTime) }}
                   </span>
                   <span v-else>Pending</span>
                 </td>
-                <td v-if="isStaff">
-                  {{ item.username }}
-                </td>
-                <td v-else>
-                  {{ item.searchQuery.clientReferenceId || '-' }}
+                <td>
+                  <span class="aria-label-only"
+                    aria-hidden="false"
+                  >{{ headers[4].text }}</span>
+                  {{ isStaff ? item.username : item.searchQuery.clientReferenceId || '-' }}
                 </td>
                 <td>
+                  <span class="aria-label-only"
+                    aria-hidden="false"
+                  >{{ headers[5].text }}</span>
```

Comment:
Same as above, some spacing and tabbing, here is an example (basically elements with multiple properties are new line each):

<img width="408" alt="Screenshot 2024-09-05 at 4 05 59 PM" src="https://github.com/user-attachments/assets/30d802bd-77bc-4368-a3e5-bff4ffea2802">